devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/examples/spring-web-example/codestart.yml

```diff
@@ -0,0 +1,17 @@
+name: spring-web-example
+ref: spring-web
+type: code
+tags: example
+language:
+  base:
+    data:
+      resource:
+        class-name: ExampleController
+        path: "/springweb/hello"
+        response: "hello"
+      package-name: org.acme.spring.web
+    dependencies:
+      - io.quarkus:quarkus-resteasy
```

Comment:
resteasy can be removed, quarkus-spring-web already depends on it.

Reply:
:grimacing: